src/main/java/auto_ckz/config/JpaConfigForTest.java

```diff
@@ -0,0 +1,71 @@
+package auto_ckz.config;
+
+import auto_ckz.Application;
+import com.zaxxer.hikari.HikariConfig;
+import com.zaxxer.hikari.HikariDataSource;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.data.jpa.convert.threeten.Jsr310JpaConverters;
+import org.springframework.data.jpa.repository.config.EnableJpaRepositories;
+import org.springframework.orm.jpa.JpaTransactionManager;
+import org.springframework.orm.jpa.LocalContainerEntityManagerFactoryBean;
+import org.springframework.orm.jpa.vendor.HibernateJpaVendorAdapter;
+import org.springframework.transaction.PlatformTransactionManager;
+import org.springframework.transaction.annotation.EnableTransactionManagement;
+import org.springframework.util.ClassUtils;
+
+import javax.persistence.EntityManagerFactory;
+import javax.sql.DataSource;
+import java.util.Properties;
+
+
+@Configuration
+@EnableTransactionManagement
+@EnableJpaRepositories(basePackageClasses = Application.class)
+
+public class JpaConfigForTest {
```

Comment:
Ta konfiguracja https://github.com/auto-ckz/auto-ckz/blob/master/src/test/java/auto_ckz/config/EmbeddedDataSourceConfig.java powinna wylecieć, tak żeby wszystkie testy korzystały z tej samej konfiguracji. Ale można to zmienić przy innej okazji.